deployments/sdm-relay/README.md

```diff
@@ -20,7 +20,7 @@ This repo provides an implementation of a strongDM relay or gateway inside Kuber
 
 * If you are going to use [Nginx Ingress Controller](https://kubernetes.github.io/ingress-nginx/), then you will need to manually patch your [services to allow TCP and UDP traffic](https://kubernetes.github.io/ingress-nginx/user-guide/exposing-tcp-udp-services/)
 
-* [A strongDM Gateway/Relay Token](https://www.strongdm.com/docs/admin-ui-guide/network/gateways)
+* Either a [strongDM Gateway/Relay Token](https://www.strongdm.com/docs/admin-ui-guide/network/gateways) or else an admin token with the `relay:create` permission which will be used to generate the gateway/relay token.

```

Comment:
should we also drop a link to the [admin token docs](https://www.strongdm.com/docs/admin/users/admin-tokens/) like we do for relay tokens?